daisymfcgui/org.daisy.pipeline.lite/.classpath

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would drop the Eclipse files.

Reply:
Fixed when Nico fixed up commit https://github.com/daisy/pipeline1/commit/fba71f39fd7f4487978529b8c5bbd76730aa62f2 to https://github.com/daisy/pipeline1/commit/736915089efcc8f4e8a9a7c0c7885fe2746bda02.